rust/ql/lib/codeql/rust/security/CleartextStorageDatabaseExtensions.qll

```diff
@@ -0,0 +1,48 @@
+/**
+ * Provides classes and predicates for reasoning about cleartext storage
+ * of sensitive information in a database.
+ */
+
+import rust
+private import codeql.rust.dataflow.DataFlow
+private import codeql.rust.dataflow.internal.DataFlowImpl
+private import codeql.rust.security.SensitiveData
+private import codeql.rust.Concepts
+
+/**
+ * Provides default sources, sinks and barriers for detecting cleartext storage
```

Comment:
nit: Oxford comma

Reply:
I thought no comma here was the standard.  Happy to be corrected (in which case there are at least 10 cases of this to update).